apps/emqx/test/emqx_shared_sub_SUITE.erl

```diff
@@ -817,6 +818,77 @@ t_dispatch_qos0(Config) when is_list(Config) ->
     emqtt:stop(ConnPid2),
     ok.
 
+t_redispatch_when_kicked({init, Config}) when is_list(Config) ->
```

Comment:
ensure snabbkaffe is stopped ? ottherwise you may get trace log from other test case.